.github/workflows/indent.yml

```diff
@@ -16,6 +16,9 @@ jobs:
     name: indent
     runs-on: [ubuntu-20.04]
 
+    # only run on 'push' or if the 'pull_request' is not a draft:
```

Comment:
Maybe we should always run indent?

Reply:
Agreed, indent alone would be fast enough.